src/pages/UploadValidator/validateDepositKey.ts

```diff
@@ -4,8 +4,49 @@
 import _every from 'lodash/every';
 import { initBLS } from '@chainsafe/bls';
 import { verifySignature } from '../../utils/verifySignature';
+import { verifyDepositRoots } from '../../utils/SSZ';
 import { DepositKeyInterface } from '../../store/reducers';
 
+const validateFieldFormatting = (
+  depositDatum: DepositKeyInterface
+): boolean => {
+  // check existence of required keys
+  if (
+    !depositDatum.pubkey ||
+    !depositDatum.withdrawal_credentials ||
+    !depositDatum.amount ||
+    !depositDatum.signature ||
+    !depositDatum.deposit_message_root ||
+    !depositDatum.deposit_data_root
```

Comment:
Field names inconsistent

1. launchpad's `deposit_message_root` is deposit-cli's `deposit_data_root`
2. launchpad's `deposit_data_root` is deposit-cli's `signed_deposit_data_root`

I'd suggest we modify deposit-cli side.

Reply:
Resolving, see: https://github.com/ethereum/eth2.0-deposit-cli/pull/32/commits/dbac6d236b1627221d8339f2ff8ec1d9ba1cb500